ReactWindows/ReactNative.Shared/Modules/Storage/AsyncStorageHelpers.cs

```diff
@@ -157,5 +158,14 @@ public static JObject GetInvalidValueError(string key)
         {
             return GetError(key, "Invalid Value");
         }
+
+        public static JObject GetError(Exception ex)
+        {
+            return new JObject
+            {
+                { "message", ex.Message },
+                { "key", ex.GetType().Name },
```

Comment:
would it be possible to stash the stack trace in here as well? then the JS promise rejection would have more remediation information in it.